packages/components/dropdown/src/use-dropdown.ts

```diff
@@ -127,7 +127,7 @@ export function useDropdown(props: UseDropdownProps) {
     const {onKeyDown, onPress, onPressStart, ...otherMenuTriggerProps} = menuTriggerProps;
 
     return {
-      ...mergeProps(otherMenuTriggerProps, {isDisabled}, originalProps),
+      ...mergeProps(otherMenuTriggerProps, {isDisabled: props.isDisabled}, originalProps),
```

Comment:
@kuri-sun the `isDisabled` is already being extracted from the `props` please check this line https://github.com/nextui-org/nextui/pull/2451/files#diff-88cdde9752c6237b962186b11532b521dd8ed7fd98b684e90b050993124cf2c9R53

Reply:
@jrgarciadev and @wingkwong 
When the `isDisabled` does not get passed to the Dropdown component, it always becomes `isDisabled=false`(https://github.com/nextui-org/nextui/blob/9b27da544e950cdb2bccd2d1a57ead772fd10dc1/packages/components/dropdown/src/use-dropdown.ts#L53). 
So I was worrying about this situation: the trigger Button component was disabled however the still is enabled.

1.  `isDisabled` (Current)

https://github.com/nextui-org/nextui/assets/62743644/58d7a3ea-f9f5-4dc7-b053-d87a2c90b8dd

3. `props.isDisabled` (This one)

https://github.com/nextui-org/nextui/assets/62743644/a763a604-d2a3-45e2-91f0-872d11c6b209



By setting `{isDisabled: props.isDisabled}`, it becomes `isDisabled: undefined`, so that coming the Button's `isDisabled` can override that, if there is any `isDisabled` on the Button component.

However, if we can ignore the Button component's `isDisabled` in this case, I would agree with going with it as it is! 😄 

Sorry for the long explanation... I hope this makes sense! Thank you! ❤️

Reply:
Okay. I finally got what you mean. I can see why we need `undefined` here now. It wasn't that obvious though. @jrgarciadev Should we cover this case as well?

Reply:
Hey guys, I fixed it here https://github.com/nextui-org/nextui/pull/2452/files